system-parachains/collectives/collectives-polkadot/src/lib.rs

```diff
@@ -381,6 +383,28 @@ impl parachain_info::Config for Runtime {}
 
 impl cumulus_pallet_aura_ext::Config for Runtime {}
 
+parameter_types! {
+	/// The asset ID for the asset that we use to pay for message delivery fees.
+	pub FeeAssetId: AssetId = Concrete(xcm_config::DotLocation::get());
+	/// The base fee for the message delivery fees.
+	pub const ToSiblingBaseDeliveryFee: u128 = CENTS.saturating_mul(3); // TODO: proper price
+	pub const ToParentBaseDeliveryFee: u128 = CENTS.saturating_mul(3); // TODO: proper price
```

Comment:
If we want to keep this TODO, we should make an issue and ensure the current value is at least safe.